reconcile/dashdotdb_dvo.py

```diff
@@ -23,29 +23,48 @@ def __init__(self, dry_run, thread_pool_size):
         self.dashdotdb_url = secret_content['url']
         self.dashdotdb_user = secret_content['username']
         self.dashdotdb_pass = secret_content['password']
+        self.chunksize = secret_content.get('chunksize') or '20'
         self.logmarker = "DDDB_DVO:"
 
+    def _chunkify(self, data, size):
```

Comment:
... but here: `if not size: size = len(data)`.

Reply:
or, even better: `if not size: return data`

Reply:
size is the max number of elements we want in a list. the typical input looks something like this:

```
data [
     resulttype.vector
     result[0].metric
     result[0].value
     result[1].metric
     result[1].value
     ...
     result[n].metric
     result[n].value
]
```
what this does it keep data.result[n] from being more than chunksize. we replicate all of result[n] to a list of dict (metrics)  where n<chunksize.. we then blow away the original results keeping the rest of the blob, and iterate on the list (metrics) for the post.

hrm. it shouldn't be necessary since the first iteration of the chunk function wouldn't satsify, but i'll add an if to skip running the chunk function if the data we want to chunk up is already less than or equal to the max desired size.